.grunt-config/sass.js

```diff
@@ -79,11 +79,18 @@ const sass = {
 			},
 			{
 				expand: true,
-				cwd: 'assets/dev/scss/frontend',
+				cwd: 'assets/dev/scss/frontend/conditionals',
```

Comment:
I am trying to group all conditionally loaded styling in one place.

I am not sure about the folder name.

Reply:
I just hope no 3rd parties tried to import `_shapes.scss` directly into their plugins.